README.md

```diff
@@ -7,6 +7,20 @@ crowdsourcing workers, and from those judgments, to compute relevance metrics
 such as normalized discounted cumulative gain (NDCG) and mean average precision
 (MAP).
 
+## Running tests
+
+From a fresh arcs virtual environment, install pytest:
+
+```bash
+pip install pytest
```

Comment:
Oooh, I didn't know about `tests_require`! You learn something new every day. :+1: